tools/validate_configs/test_configs/config-ssh.yaml

```diff
@@ -31,12 +31,12 @@ deployment_groups:
   # Source is an embedded resource, denoted by "resources/*" without ./, ../, /
```

Comment:
nit. "resource" is deprecated word, replaced by "module". Pls, either update the comment or remove it all together.

Reply:
never mind, I've addressed it in #2364